scss/_patterns_tooltips.scss

```diff
@@ -53,6 +53,16 @@ $triangle-height: $sp-unit;
       position: absolute;
       width: 0;
     }
+
+    &::after {
```

Comment:
As these pseudo-elements (both `::before` and `::after`) don't have any classnames that could describe their meaning it would be good to put a comment above each of those to describe their use.

So `::before` would be the tooltip triangle 'tip', and `::after` a hover overlay?